youtube_dl/extractor/tvp.py

```diff
@@ -15,16 +15,20 @@
 class TVPIE(InfoExtractor):
     IE_NAME = 'tvp'
     IE_DESC = 'Telewizja Polska'
-    _VALID_URL = r'https?://[^/]+\.tvp\.(?:pl|info)/(?:(?!\d+/)[^/]+/)*(?P<id>\d+)'
+    # vod.tvp.pl has changed the URL format thus the below regular expression is a combination of the old
+    # format r'https?://[^/]+\.tvp\.(?:pl|info)/(?:(?!\d+/)[^/]+/)*(?P<id>\d+)' (still used by other
+    # subdomains of tvp.pl) and the new format r'https?://[^/]+\.tvp\.(?:pl|info)/video/(?:[^,\s]*,)*(?P<id>\d+)'
+    # Regular expression's test page: https://regex101.com/r/4iu6Hv/1/
```

Comment:
This can be figured out from git log and diff thus should be removed.

Reply:
Removed.